packages/contracts-bedrock/src/L2/ETHLiquidity.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
```

Comment:
We generally have been using `0.8.25` for new contracts, while this one doesn't use 1153, we do want to generally update all contracts to more modern solc

Reply:
As per external convo, have decided to stick with 0.8.15 for now and will make a more concerted push to move to 0.8.25 separately.